spec/javascripts/components/story/story_popover_spec.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+import moment from 'moment';
+import StoryPopover, {
+  StoryPopoverContent
+} from 'components/story/StoryPopover';
+
+describe("<StoryPopover />",() => {
+
+  beforeEach(function() {
+    sinon.stub(I18n, 'translate');
+  });
+
+  afterEach(function() {
+    I18n.translate.restore();
+  });
+
+  it("renders the popover",() => {
+    const props = {
+      title: 'title',
+      description: 'description',
+      storyType: 'feature',
+      requestedByName: 'Foo Bar',
+      createdAt: '2018/10/26 15:08:51 -0300',
+      notes: []
+    };
+    const wrapper = shallow(<StoryPopover {...props} />);
+    expect(wrapper.find("Popover").props().title).toEqual(`${props.title}`);
```

Comment:
You can make it shorter like this (https://github.com/FormidableLabs/enzyme-matchers/tree/master/packages/jasmine-enzyme#tohaveprop)

```suggestion
    expect(wrapper.find("Popover")).toHaveProp('title', props.title);
```